config/install/editor.editor.full_html.yml

```diff
@@ -1,10 +1,13 @@
+uuid: c8b180c0-46de-40dd-b717-1b0058af40ff
```

Comment:
We need to manually remove the UUID line because this will be used in a distribution instead of individual site.

Reply:
Hi Michael
This has been fixed now